.github/workflows/shopify-cli-extensions-release.yml

```diff
@@ -6,7 +6,13 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  ARGO_EXTENSIONS_DIR: packages/extensions/shopify-cli-extensions
+
 jobs:
+  defaults:
+    run:
+      working-directory: "$env.ARGO_EXTENSIONS_DIR"
```

Comment:
I'm curious why we need to set this to an env variable?

Reply:
Probably this variable can also be reused within the makefile to avoid hardcoding the path the different steps.